### PR TITLE
Fix pod logs (and dynamically updating the log level in general)

### PR DESCRIPTION
**Pull request by scme0**
# Background

My previous attempt at add pod logs doesn't seem to work properly. I'm not sure how that slipped by but anyways, this seems to fix it.

# How to review this PR

Quality :heavy_check_mark:

# Pre-requisites

- [ ] I have read [How we use GitHub Issues](https://github.com/OctopusDeploy/Issues/blob/main/docs/CONTRIBUTING.internal.md) for help deciding when and where it's appropriate to make an issue.
- [ ] I have considered informing or consulting the right people, according to the [ownership map](https://whimsical.com/ownership-map-NzbiD4HJyvhC9jNJNfS6TG).
- [ ] I have considered appropriate testing for my change.

### CODE REVIEW

File: source/Octopus.Tentacle/Tentacle.exe.nlog [hunk 1]
```diff
@@ -12,7 +12,7 @@
   </extensions>
 
   <variable name="appName" value="Octopus Tentacle" />
-  <variable name="messageLayout" value="${message}${onexception:${newline}${exception:format=ToString}}"/>
+  <variable name="messageLayout" value="${longdate} | ${message}${onexception:${newline}${exception:format=ToString}}"/>
```
Comment (flin-8) [inline source/Octopus.Tentacle/Tentacle.exe.nlog:15]: Do we need to make this change? Lens has a way to show timestamps

CleanShot 2024-02-20 at 16 55 35@2x

Comment (scme0) [reply source/Octopus.Tentacle/Tentacle.exe.nlog:15]: Oh I didn't know! Thanks, I'll revert this change then 👍

Comment (scme0) [reply source/Octopus.Tentacle/Tentacle.exe.nlog]: Done 👍

**Review by flin-8 — APPROVED** [review]
Nice!

**Pull request merged**